lib/Extended_ast.ml

```diff
@@ -101,6 +107,28 @@ module Parse = struct
 
   let normalize fg x = map fg fix_letop_locs @@ map fg normalize_lists @@ x
 
+  let repl_file (lx : Lexing.lexbuf) =
+    let str = String.strip (Bytes.to_string lx.lex_buffer) in
```

Comment:
This only works by chance and might break in the future because that's not how this API is supposed to be used.
I think we should use Mdx's lexer directly and remove this potential bug and the string manipulations. https://github.com/realworldocaml/mdx/blob/1.11.1/lib/lexer_top.mll

Reply:
done